.travis.yml

```diff
@@ -45,6 +47,10 @@ script: |
       bash -ec 'source loadLSST.bash && echo $EUPS_PKGROOT'
       ;;
   esac
+
+  echo -e "*** Sanity checking EUPS_PKGROOT construction ***\n"
+  source loadLSST.bash
+  eups distrib install cfitsio -t v14_0_rc2
```

Comment:
given our lsstsw problems with building cfitsio maybe you should use `python` here as that is the simplest possible eups distrib package to install.

Reply:
That sounds reasonable.  I'll amend the PR.

Reply:
Actually, `astropy` is probably a better package to test against. That one can't work unless astropy is installed and there is no way that astropy is installed on travis (whereas the python that is installed will pass the stub even if the wrong python is used). This failure mode was spotted on Slack just now.